addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -142,12 +142,12 @@ export class Powerbox {
 
     addKeydownTrigger(triggerKey, options) {
         this.options.editable.addEventListener(
-            'keydown',
+            'input',
```

Comment:
You should rename the function to match the change of event. I'm just wondering though if we do that if we can do it in stable since it might qualify as a change of API...
If not then don't bother doing it here but it should be done in master.

Did you test that it works properly with all keyboards (swiftkey?) and on all major browsers?